generator/internal/format_method_comments.cc

```diff
@@ -37,6 +37,81 @@ ProtoDefinitionLocation Location(google::protobuf::Descriptor const& d) {
   return ProtoDefinitionLocation{d.file()->name(), loc.start_line + 1};
 }
 
+std::string ReturnCommentString(
+    google::protobuf::MethodDescriptor const& method) {
+  if (IsLongrunningOperation(method)) {
+    return R"""(  /// @return A [`future`] that becomes satisfied when the LRO
+  ///     ([Long Running Operation]) completes or the polling policy in effect
+  ///     for this call is exhausted. The future is satisfied with an error if
+  ///     the LRO completes with an error or the polling policy is exhausted.
+  ///     In this case the [`StatusOr`] returned by the future contains the
+  ///     error. If the LRO completes successfully the value of the future
+  ///     contains the LRO's result. For this RPC the result is a
+  ///     [$longrunning_deduced_response_message_type$] proto message.
+  ///     The C++ class representing this message is created by Protobuf, using
+  ///     the [Protobuf mapping rules].
+)""";
+  }
+  if (IsBidirStreaming(method)) {
+    return absl::StrFormat(
+        R"""(  /// @return An object representing the bidirectional streaming
+  ///     RPC. Applications can send multiple request messages and receive
+  ///     multiple response messages through this API. Bidirectional streaming
+  ///     RPCs can impose restrictions on the sequence of request and response
+  ///     messages. Please consult the service documentation for details.
+  ///     The request message type ([%s]) and response messages
+  ///     ([%s]) are mapped to C++ classes using the
```

Comment:
Should the type names be back-tick quoted?

Reply:
I think so.  I even implemented the fix, but it broke other links extracted from the proto comments.  I created #11685 to track this.